first-year-scraper/requirements.txt

```diff
@@ -0,0 +1,5 @@
+camelot-py>=0.11.0
+opencv-python
+ghostscript
```

Comment:
Is this dependency used anywhere?

Reply:
Yes, by camelot-py.

Reply:
Wouldn't pip install all sub-dependencies? Do we need to specify it? On my system it worked without installing ghostscript.

Reply:
Although I did have to install the _system_ package ghostscript.

Reply:
> Wouldn't pip install all sub-dependencies?
 
It did not for some reason. 
What I did to find all dependencies was create a new conda environment with python=3.8, then I ran the script and added dependencies as I got errors of missing packages.

Reply:
Weird. But it's Python so we can't question it.